packages/angular_devkit/build_angular/src/testing/jasmine-helpers.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { BuilderHandlerFn } from '@angular-devkit/architect';
+import { json } from '@angular-devkit/core';
+import { readFileSync } from 'fs';
+import { host } from '../test-utils';
+import { BuilderHarness } from './builder-harness';
+
+export function describeBuilder<T>(
+  builderHandler: BuilderHandlerFn<T & json.JsonObject>,
+  options: { name?: string; schemaPath: string },
+  specDefinitions: (harness: JasmineBuilderHarness<T>) => void,
+): void {
+  const optionSchema = JSON.parse(readFileSync(options.schemaPath, 'utf8'));
```

Comment:
Wouldn't it be better to use `require()` so that the schema is cached and avoid an extra IO for every spec file?

I know it won't make much difference between the underlying tests still use the IO heavily.

Reply:
Cached the schemas.  I generally prefer not to use `require` for data files.  We also may want to use the full JSON parser instead of `JSON.parse` in the future.

Reply:
Fair enough